omod/src/main/webapp/fragments/field/concept.gsp

```diff
@@ -0,0 +1,41 @@
+<%
+	import groovy.json.*
+    config.require("formFieldName")
+    config.require("conceptId")
+    
+    def maxResults = config.maxResults ? config.maxResults : 20;
+    def conceptAnswers = new JsonBuilder(conceptAnswers);
+%>
+
+<script type="text/javascript">
+    jq(function() {
+    var consAnswers = ${conceptAnswers};
+        jq('#${ config.id }-field').autocomplete({
```

Comment:
I thought you mentioned that you only wanted to use the autocomplete if the list is longer than the maxResults otherwise it would be a dropdown

Reply:
Yes, but now the structure was changed and i guess there's no need to separate it for autocomplete and dropdown as anyway it will give us the full list of answers as dropdown if the length is less than the maxResults

Reply:
If it's an autocomplete you need to show all results